hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/impl/NetworkUtilizationManagerImpl.java

```diff
@@ -165,10 +166,10 @@ public void saveTo(@NonNull final HederaState state) {
         final var congestionLevelStartsState =
                 states.<CongestionLevelStarts>getSingleton(CongestionThrottleService.CONGESTION_LEVEL_STARTS_STATE_KEY);
         final var genericCongestionStarts = Arrays.stream(multiplierSources.genericCongestionStarts())
-                .map(inst -> new Timestamp(inst.getEpochSecond(), inst.getNano()))
+                .map(inst -> inst == null ? null : new Timestamp(inst.getEpochSecond(), inst.getNano()))
```

Comment:
Can we be sure that all code can deal with `null` in the `List`? Should we filter `null`s?

Reply:
Yes, actually we can just filter the `null`s, will update that.